Gemfile

```diff
@@ -62,9 +62,14 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
+  gem 'rails-controller-testing'
   gem 'selenium-webdriver'
+  gem 'simplecov'
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
+  %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
```

Comment:
Remind me why did you need to install github version and not rubygems.org?
also, you have git_source helper defined in line 4, google how to use it.
Also never specify master branch as it may be unstable.
Also use git repositories carefully because of security implications.

Reply:
Remind me why did you need to install github version and not rubygems.org? - Now I know, 
don't do that

by the way, you saw this code in the camp and didn’t tell me anything) 

done